components/mdx/Code.tsx

```diff
@@ -74,7 +74,7 @@ export default function Code({
     breakWords = true;
   }
 
-  const hasCopy = props.copy || language === "copy";
+  const hasCopy = /*props.copy || language === "copy"*/ true;
```

Comment:
Do we want everything to be copyable? Wondering if it makes more sense to make this opt-in instead

(For opt in, you would do this:

```
```json copy
code
```

Reply:
Think it would make sense for it to be opt-out rather than opt-in.

Reply:
If we were to go with a make every languaged code copyable route, some stuff like URLs and weird examples wouldn't be copyable.
If we were to go with an opt in version, we'd have to update the docs a ton
If we were to go with an opt out version, we'd still have to update the docs, but it might be the easiest method.

Reply:
I think opt in is better, most of the docs don't need to be copyable (e.g.  example objects)